web/src/components/vis/VisTile.vue

```diff
@@ -11,28 +11,61 @@ export default {
     },
   },
 
+  data() {
+    return {
+      scale: 1,
+    };
+  },
+
   computed: {
-    hasControls() {
-      return !!this.$slots.controls;
+    smallEnough() {
+      return this.scale <= 0.5;
+    },
+    largeEnough() {
+      return this.scale >= 4;
+    },
+    scaleClass() {
+      return `scale${Math.round(this.scale * 100)}`;
```

Comment:
`* 100` seems unnecessary.  Why not just have your min and max be 50 and 400 respectively?

Actually, why was multiplication used here? if the point is to have N different classes which apply N different grid sizes, why not just use addition?  `scale += 1` and `scale -= 1` would achieve the same thing without the risk of floating point issues.  You're already stuck on an exponential scale, so it's not like you could add scale=500 if you wanted something in the middle anyway.

Reply:
Move this conversation to the thread below.